subcommands/targets/offline-update.go

```diff
@@ -87,6 +89,8 @@ func init() {
 		"Skip fetching Target Apps")
 	offlineUpdateCmd.Flags().BoolVarP(&ouAllowMultipleTargets, "allow-multiple-targets", "", false,
 		"Allow multiple targets to be stored in the same <dst> directory")
+	offlineUpdateCmd.Flags().StringVarP(&ouOstreeRepoSrc, "ostree-repo-source", "", "",
+		"Path to the local ostree repo to be used in the offline bundle")
```

Comment:
Maybe "to be added to the offline bundle" or "to be copied to"?

Reply:
Now using `to be added to`